example.json

```diff
@@ -22,6 +22,7 @@
             "_type":"document",
             "title":"Button idea",
             "description":"Click here to see a proposed button idea",
+            "filename":"Buttons.pdf",
```

Comment:
Maybe call it `_filename`, as underscores are generally used to indicate properties which are not translatable